components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -6812,7 +6812,12 @@ public static void logAuditMessage(String entityType, String entityInfo, String
         jsonObject.put("typ", entityType);
         jsonObject.put("action", action);
         jsonObject.put("performedBy", performedBy);
-        jsonObject.put("info", entityInfo);
+        try {
+            JSONObject entityInfoJson = (JSONObject) new JSONParser().parse(entityInfo);
```

Comment:
Have we convert the entityInfoJson into a JSONObject? can see the type as string in here: https://github.com/wso2/carbon-apimgt/blob/9.22.x/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java#L6809